glanzb/homework1.md

```diff
@@ -0,0 +1,183 @@
+### Homework #1
+(Due Monday 1/12)
+
+You need to turn in this homework by cloning this repo, making a new branch with your solutions, pushing to github, and issuing a pull request.
+For details, follow the instructions [here](http://portlandcodeschool.github.io/jse/2015/01/07/command-line-and-git-slides/#/14).
+
+Please also read the [learning objectives](objectives.md) for this week.
+
+---
+
+**1)** (_Difficulty: mixed_)
+Predict the output of each of the following expressions, and explain the rules which determine it.
+If the output depends on the value of x, identify the conditions when the expression will output true.  Assume the cases are independent, and x is reset to an unknown value before each.
+
+Some of these are tricky!  Don't trust your first instinct.  
+
+
+**a)** `"1" == 1`
+	true, number and string 1 are similar enough, js vill convert the string into number
+
+**b)** `"1" === 1`	->	false, string 1 and number one are not equal
+
```

Comment:
you shouldn't be getting  x is undefined 
make sure you are testing things like this:

var x; console.log(x == ('x'));

where you are defining x and then using it.